config/routes.rb

```diff
@@ -8,7 +8,7 @@
   #get '/merchants/:id/items/:id', to: 'items#edit'
   resources :admin, only: [:index]
   namespace :admin do
-    resources :invoices, only: [:index, :show, :update]
-    resources :merchants, only: [:index, :show, :edit, :update]
+    resources :invoices, only: [:index, :show]
+    resources :merchants, only: [:index, :show, :edit, :update, :new, :create]
```

Comment:
I think at this point you can remove the only as all the actions have now been added.

Reply:
Let me do this now!